specs/pvs/corla.pvs

```diff
@@ -1053,9 +1136,9 @@ BEGIN
 %%% ballot polling audits, but in no event will the risk limit exceed
```

Comment:
why mention ballot polling audits in the spec, since the system doesn't support ballot polling audits? If we're mentioning them for readers such as CORLA2 or SAWG members, we could put them into some bit of prose that's not buried in the spec details.